tests/Resources/ti.ui.webview.test.js

```diff
@@ -14,6 +15,7 @@ describe('Titanium.UI.WebView', function () {
 	this.slow(3000);
 	this.timeout(30000);
 
+	const isiOS14 =  (parseInt(Ti.Platform.version.split('.')[0]) >= 14);
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.webview.test.js;
  Line: 18;
-->

- :warning: tests/Resources/ti.ui.webview.test.js line 18 – 'isiOS14' is assigned a value but never used. (no-unused-vars)

Reply:
Remove this line